package.json

```diff
@@ -21,5 +21,8 @@
     },
     "resolutions": {
         "**/**/lodash": "^4.17.19"
+    },
+    "dependencies": {
+        "lodash": "^4.17.19"
```

Comment:
As we only seem to use the npm packages in dev mode, it might be a good idea to put this lodash version pin in the `devDependencies` section? I also see this same version already in the resolutions. So maybe you are missing a npm 'fix resolutions' step, and thats why you want to version pin it?

See `https://www.npmjs.com/package/npm-force-resolutions` for details on how to install the resolutions